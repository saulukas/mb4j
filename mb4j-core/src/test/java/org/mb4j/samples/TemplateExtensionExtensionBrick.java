package org.mb4j.samples;

public class TemplateExtensionExtensionBrick extends TemplateExtensionBrick {
  public TemplateExtensionExtensionBrick() {
    this.header = new TemplateExtensionExtensionHeaderBrick();
  }
}
