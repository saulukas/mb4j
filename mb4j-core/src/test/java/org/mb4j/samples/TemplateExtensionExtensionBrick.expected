
   Header: Nice and Lighweight web-app with clean HTML source

   ....
   ....   Some content from extending Brick.
   ....
   ....   Kind of removes need for separate layout/tiles mechanism.
   ....

   Footer: Â© 2013
